integration/aws_cloud_test.go

```diff
@@ -53,7 +50,8 @@ func TestAwsCommandRun(t *testing.T) {
 
 	ctx := context.Background()
 
-	localstackC, addr := setupLocalStack(t, ctx)
+	localstackC, addr, err := testutil.SetupLocalStack(ctx, "2.2.0")
```

Comment:
You don't disable `RYUK` for this test now. 
Is this expected?

Reply:
Fixed https://github.com/aquasecurity/trivy/pull/6176/commits/686d2275aa9c72248587ba16475f6121abbaaa39